koku/masu/database/azure_report_db_accessor.py

```diff
@@ -164,3 +165,11 @@ def populate_markup_cost(self, markup, bill_ids=None):
             else:
                 AzureCostEntryLineItemDailySummary.objects.\
                     update(markup_cost=(F('pretax_cost') * markup))
+
+    def get_lineitem_query_for_billid(self, bill_id):
+        """Get the Azure cost entry line item for a given bill query."""
+        table_name = AzureCostEntryLineItemDaily
```

Comment:
can we make this `get_daily_query_for_billid`?
In AWS, we have:
```
get_lineitem_query_for_billid:
    table_name = AWSCostEntryLineItem
get_daily_query_for_billid:
    table_name = AWSCostEntryLineItemDaily
```

Reply:
Agreed. :+1:

Reply:
So this is where Azure departs from AWS and OCP.  For Azure the line item is already daily (We don't have a separate Daily table for Azure).  I made this line item to be more explicit that it's the line item data (and it just happens to have a Daily the table name to make it clear that it's daily data.  Obviously the naming conventions of the tables was not successful :) But hopefully that makes sense.  Let me know if it doesn't